scipy/signal/sigtoolsmodule.c

```diff
@@ -1242,9 +1242,7 @@ static PyObject *sigtools_remez(PyObject *NPY_UNUSED(dummy), PyObject *args) {
                       type, maxiter, grid_density);
         if (err < 0) {
 	  if (err == -1) {
-            sprintf(mystr, "Failure to converge after %d iterations.\n      " \
-                           "Design may still be correct.",
-                    maxiter);
+            printf(mystr, "Failure to converge after %d iterations.\n", maxiter);
```

Comment:
the change from ``sprintf`` to ``printf`` isn't right. the message isn't printed to stdout but stored in ``mystr`` and used in the line below